creator-node/src/components/replicaSet/URSMRegistrationComponentService.js

```diff
@@ -136,16 +136,20 @@ const respondToURSMRequestForSignature = async ({ libs: audiusLibs, nodeConfig }
    * Confirm health check response was signed by delegate owner wallet registered on L1
    *    for spID and includes `randomBytesToSign`
    */
-  let {
-    timestamp: respTimestamp,
-    signature: respSignature,
-    ...responseData
-  } = nodeHealthCheckResp.rawResponse
-  if (signatureHasExpired(respTimestamp)) {
+  const {
+    timestamp,
+    signature,
+    signer
+  } = signatureData
+  if (signatureHasExpired(timestamp)) {
     throw new ErrorBadRequest('Health check response signature has expired')
   }
-  const responderWalletRecoveryObj = { ...responseData, randomBytesToSign, timestamp: respTimestamp }
-  const recoveredDelegateOwnerWallet2 = (recoverWallet(responderWalletRecoveryObj, respSignature)).toLowerCase()
+  const responderWalletRecoveryObj = {
+    data: responseData,
+    signer,
+    timestamp
+  }
```

Comment:
curious why you changed this from `randomBytesToSign` key to `signer`?
looks like the `randomBytesToSign` field is actually redundant since its already present in `responseData` so this change makes sense, just confirming

Reply:
^ i believe this is correct as well. previously it was just overwriting the existing property in the responseData object

Reply:
yep it was already there!